src/data/supervisor/supervisor.ts

```diff
@@ -82,7 +82,7 @@ async function processEvent(
   event: SupervisorEvent,
   key: string
 ) {
-  if (event.event !== "supervisor-update" || event.update_key !== key) {
+  if (event.event !== "supervisor_update" || event.update_key !== key) {
```

Comment:
Not really related to the description of the PR, but this was also wrong and probably has never worked...
<https://github.com/home-assistant/supervisor/blob/main/supervisor/homeassistant/websocket.py#L256>
<https://github.com/home-assistant/supervisor/blob/main/supervisor/homeassistant/const.py#L30>